### ISSUE TITLE
Issue #26 - Do we really need the Social Status pattern?

### ISSUE THREAD

**Issue by stephenhart8** [chin-rcip/chin-rcip#27]
The social status pattern had the purpose of modeling specific kinds of occupation, like being a student or being the elder of an indigenous community. I was never really happy with this pattern as I created it only to handle those two special cases.

It is most possible that studentship will be modeled as being a member of the University, in the role of student (see issue [#11](https://github.com/chin-rcip/chin-rcip/issues/11)).

Therefore, this Social Status pattern only models occupations like Elder. Can we not fit those kind of role/occupation somewhere else in the TM? Or would there be other kind of occupation that would justify to keep this pattern?

**Comment by KarineLeonardBrouillet** [comment]
I think it depends but generally speaking status is not necessarily linked to occupations as much as positioning in a (somewhat) hierarchical stratification of society. I think there are other examples such as leaders, experts, gurus, etc. These are statuses, just like that of elders, that are conferred on someone by their social group and are reliant on recognition rather than one's occupation.

**Comment by stephenhart8** [comment]
Indeed, like someone would be knighted, it would be modeled in the Social Status?
If we all agree, I can keep the Social Status in the TM with some examples and close the issue.

**Comment by Habennin** [comment]
+1 to keeping social status

**Comment by KarineLeonardBrouillet** [comment]
I stand by my position that it has to remain, although I am not sure how knighthood should be modeled. Assuming you are referring to modern knighthood (medieval knighthood could be considered a military occupation I think), it is first and foremost a title that confers prestige and thus status in a way I suppose. However, I am not sure how we should treat such data: should a title automatically generate a status and vice versa (e.g. Knight Commander of the Most Excellent Order of the British Empire as status leads to KBE and Sir as affixes to the name)?

**Comment by stephenhart8** [comment]
That is an interesting question @KarineLeonardBrouillet as if we should create prefixes to the name or not.
I took the example of Jean Paul Riopelle (as always...) in the TM for the social status.  Riopelle has been who has been an officer and then a companion of the Order of Canada.
![Example_Social_Status_Riopelle](https://user-images.githubusercontent.com/49342191/68693796-e7ea5d80-0545-11ea-8784-8102209f66b9.png)

That seems to be a good example for Social Status, but it could also be interpreted as being a member of a group, as it would be with the freemasonry? I think the criteria could be if this title leads to activities with the rest of the members (modeled as a group) or not (modeled as a social status).

**Comment by Habennin** [comment]
I am still not for these very embedded hierarchies of types... which look like SKOS to me (which is fine but then do the heavy hierarchy in your vocab).

In CRMSoc we are modelling social role as a sort of phase. You can check out all the draft material here and here:

http://ontome.dataforhistory.org/namespace/24#classes

http://www.cidoc-crm.org/crmsoc/

That being said, I think the simplest way to think of social role is being assigned a status in a society... in CRM terms this is an attribute assignment. So someone can be given a title. The mechanism is already there for modelling this. E13... Following that if you want to say that so and so used a title for such and such a period as a separate statement then you can use the FRBR pattern of name use.

ie F52

**Comment by illip** [comment]
For the embedded hierarchies, I think we should keep this discussion in issue #29. Like I said in this issue, I think a two-level hierarchy is enough in our model.

First thing first, we should probably identify clearly what we consider to be a social status. I think there is different kinds of social status that we might model differently. From my point of view, a student should be model as a member of a group (with the p144.1) because I assume they can perform collectively actions (they follow schedule, rules, etc.). For me this pattern is working very well. 

However, I don't think I would to same thing for an elder because it's less clear if they are performing things collectively. In this case, we could use the Phase pattern but I would like to wait for something more official from the CRM. 

So if I understand @Habennin 's idea we could use e13 Pattern but I'm not sure with which class. The simplest idea is probably to go with e55 but I'm not sure if it's enough for our needs:

![issue#27](https://user-images.githubusercontent.com/48293227/69486972-160e4e00-0e20-11ea-80ae-831faf50442b.jpg)

For the elder, I guess we would like to include the associated community and I'm not sure exactly how I would model it.

To sum up, I think we should identify clearly what we would like to model as a Social Status and decide if we prefer to keep it simple until we get some use cases.

**Comment by stephenhart8** [comment]
I also have one question about the `E13 Attribute Assignment` pattern: in that case of an Elder, who is the actor performing the `E13 Attribute Assignment`? Do we need one?
In the case of a knighthood, it would then be the Queen that is the actor performing the `E13 Attribute Assignment`?

Do you think @illip that we should keep a list of every use cases and how we modeled them?

**Comment by Habennin** [comment]
I, after a very long hiatus, respond to the E13 question.

In my research last summer, I was thinking a lot about the creation of social facts and looking for good conceptualizations of them. I ended up re-reading Austin and Searle and think that this gives a nice model for how to think about gaining and losing social status (amongst other social valuations). 

A social status is a reality, but it has no physical form. Nevertheless, it is real in its effects. We simply do act in a different way towards the Queen regardless of our take on the best forms of government, because we live in a society constituted to recognize the person holding the role of Queen in some particular way. This is the case so long as there is a society which has constituted a certain set of social rules and which believes in them (before chaos breaks in).

So we come to the basic question, how does this come about. This is a big part of Searle's ouevre and he bases himself on Austin. The analysis proceeds from speech act theory. Speech act theory investigates the power of 'words to do things'.  Speech act theory specifies that an event must occur that follows a rule, engages specific actors, performing certain functions, according to a specified set of actions and having the will and the capacity to follow through with their act. In such circumstances they may utter a sentence which by society now changes reality, it bestows a new value upon some real world object which now has a social function and meaning which in the past it did not have. for example getting married. Where it gets philosophically complicated is when one has to first create the rules by which the speech act will occur. But I digress.

E13 can be used as a special case of speech act. It is an event of some actor in time who is socially empowered to provide a designation/property to some other thing. In the case of knighting, it is the Queen who acts. In the case of a democracy, it could be the demos that acts. In the case of brute force, it could be the self same actor (I declare myself master of X).

**Comment by KarineLeonardBrouillet** [comment]
Notes on verbal meeting 2020-02-17

How does the e13 really work? 
Instead of having someone performing the activity of being a knight the actor would receive the attribute assignment knighthood through the e13. This documents when the attribute was made (or how long?) and who made it. There is no need to have an actor performing the attribute assignment. 

It is dissimilar to the group discussion. There has a static binary (e.g. sally has such status) and then you can ask the question since when, are they still, etc. In that case, could use attribute assignments because there is some event that caused a property to be enforced to exist. Can use assigned attributive type to point to the list of attributes. The general argument is that they are all performative rituals that create a new social circumstance based on the ritual. 
SH’s note: In the next version of CIDOC CRM, there will be the property P170 or P171 assigned the attribute of type. That will allow linking the E13 with the property it assigned.

**Comment by illip** [comment]
CHIN has decided to use both patterns (group belonging and attribute assignment). Eventually, we would like to identify clearly which activities imply the involvement of group and when an attribute assignment is preferable. 

We keep this issue open to gather more use cases.

**Comment by marielmat** [comment]
The social statuses we document in the Actor records are mostly (but not exclusively) associated with religious hierarchy. We could improve this aspect. For now, we use two fields to record the following terms:
- Honorifics: Lady, Lord, Monseigneur, Sir
- Suffix : abbé, (l')Ainé, archevêque, baron, cardinal, chanoine, comte, curé, dit(e), duc(hesse), évêque, fils, frère, junior, le jeune, le Vieux, marquis(e), père, révérend, saint, seigneur (sieur), senior, soeur

When the information is known, the association to the religious community is added after the suffix, e.g. Beatrix of Rome, soeur, s.c.q.

In the Objects/Archives records, the social statuses will generally be found in a free text field.

Depending on the donor's wishes, certain statuses may be mentioned in the Mention field. These include professional titles (e.g. docteur), religious titles (such as those mentioned above), honorary titles (e.g. Musée de la civilisation, donation of the Honourable Serge Joyal, P.C., O.C.) and titles of nobility (e.g. lady).

**Comment by Christian-McCord** [comment]
Au Musée McCord, lorsque connu, le statut social est parfois précisé dans les champs "Honorific", "Suffix", "Salutation" et "Job Title"  des enregistrements pour les Actors.  Il est également parfois précisé en tant qu'attribut dans deux différentes catégories nommée "Titles" et "Occupation", ou alors librement dans un texte biographique.  En ce qui concerne les représentations d'Actors, le statut social est parfois précisé dans le titre de l'oeuvre, ou avec différents mots-clés versés dans la zone Sujet.  On note une prédominance de termes liés à la position/fonction publique, politique ou militaire des Actors: Admiral, Captain, Lieutenant Colonel, General, Count, King, Judge, officer, Marquis, Marechal, chanceller, Bishop, Benedict, Chief Justice, Commander, Earl, Sir, Prince, Queen, Lord, Reverend, Honourable, Mother (religious), Sister (religious), Duke, Mayor, Major, Majesty, Royal Highness, Indigenous Chief, Abbé, Minister, Prime Minister, Solicitor General of Canada, President, Member of Parliament, Governor General of Canada, Executive Director, etc.

**Comment by emchateau** [comment]
Je pense que la distinction est importante. Car le statut social est ontologiquement différent de l’occupation d’un acteur. Le problème réside surtout dans le fait qu’il est parfois difficile de déterminer ce qui doit être décrit comme statut ou occupation. 

Nous avons le pb dans l’ANR Experts où les experts sont maçons ou architecte et exercent en qualité d’experts. Qualité et métiers sous l’Ancien Régime sont des notions distinctes.

Il me semble que conserver la distinction est importante pour prévoir aussi la prise en charge des besoins spécifiques relatifs aux cultures traditionnelles.

**Comment by emchateau** [comment]
`E13 Attribute Assignment` paraît adaptée mais il n’est pas actuellement possible de retirer un statut social. Lacune du modèle.

**Comment by illip** [comment]
The Semantic Committee has decided on 2021-04-01 to keep the Social Status pattern as it is something more than relevant to implement. However, a few things must be done in regard to the [current pattern](https://chin-rcip.github.io/collections-model/en/target-model/current/life-events#social-status):

1. Replace E7_Activity by E5_Event as the E39_Actor is not always actively "performing" this social status.
2. Add E13_Attribute_Assignment that will be connected to the E5_Event and the E39_Actor in order to document the attribution activity of this social status. This way we would be able to tell who gave this social status.
3. Add a temporal link between the E13_Attribute_Assignment and the E5_Event in order to document that the first one starts the second one. The property O13_triggers from CRMsci could be used to do that.
4. We will have to better define what is a Social Status, especially to make a clear distinction with Occupation.
5. Indicate that this pattern is a temporary one while we wait for the first version of CRMsoc.
